verification/curator-service/ui/src/components/landing-page/SignInForm.tsx

```diff
@@ -81,11 +81,14 @@ interface FormValues {
     isNewsletterChecked: boolean;
 }
 
-export default function SignInForm() {
+interface SignInFormProps {
+    setRegistrationScreenOn: any;
+}
+
+export default function SignInForm({setRegistrationScreenOn}:SignInFormProps) {
     const classes = useStyles();
 
     const [passwordVisible, setPasswordVisible] = useState(false);
-    const [registrationScreenOn, setRgistrationScrenOn] = useState(false);
     const [forgotPasswordScreenOn, setForgotPasswordScreenOn] = useState(false);
 
     const validationSchema = Yup.object().shape({
```

Comment:
in this component you are using only `email`, `password` and `isAgreementChecked` in your form, so you can delete the rest from formik and `validationSchema`